dipy/workflows/reconst.py

```diff
@@ -139,4 +144,100 @@ def get_fitted_tensor(data, mask, bval, bvec, b0_threshold=0):
     tenmodel = TensorModel(gtab)
     tenfit = tenmodel.fit(data, mask)
 
-    return tenfit, gtab
\ No newline at end of file
+    return tenfit, gtab
+
+
+def reconst_csd_flow(input_files, bvalues, bvectors, mask_files,
+                     b0_threshold=0.0,
+                     frf=[15.0, 4.0, 4.0], out_dir='',
+                     out_peaks='peaks.npz'):
+    """ Workflow for peaks computation. Peaks computation is done by 'globing'
+        ``input_files`` and saves the peaks in a directory specified by
+        ``out_dir``.
+
+    Parameters
+    ----------
+    input_files : string
+        Path to the input volumes. This path may contain wildcards to process
+        multiple inputs at once.
+    bvalues : string
+        Path to the bvalues files. This path may contain wildcards to use
+        multiple bvalues files at once.
+    bvectors : string
+        Path to the bvalues files. This path may contain wildcards to use
+        multiple bvalues files at once.
+    mask_files : string
+        Path to the input masks. This path may contain wildcards to use
+        multiple masks at once. (default: No mask used)
+    b0_threshold : float, optional
+        Threshold used to find b=0 directions
+    frf : tuple, optional
+        Fiber response function to me mutiplied by 10**-4 (default: 15,4,4)
+    out_dir : string, optional
+        Output directory (default input file directory)
+    out_peaks : string, optional
+        Name of the peaks volume to be saved (default 'peaks.npz')
+    """
```

Comment:
I think it is mandatory idea to manage file overwrite. Ill look into a way to integrate the parameter seamlessly to every new workflow like the logging params in flow_runner.